internal/util/errors.go

```diff
@@ -33,6 +33,9 @@ var (
 	ErrSnapNameConflict = errors.New("snapshot name conflict")
 	// ErrPoolNotFound is returned when pool is not found.
 	ErrPoolNotFound = errors.New("pool not found")
+
```

Comment:
According to the file convention, Newline is not required.

Reply:
@Madhu-1 ^^